app/main.go

```diff
@@ -30,13 +30,14 @@ import (
 
 var opts struct {
 	Telegram struct {
-		Token   string        `long:"token" env:"TOKEN" description:"telegram bot token" default:"test"`
```

Comment:
it seemed to be useless to provide defaults for telegram token, because `Do` in telegram listener returns an error when it can't get chat id